target/linux/ramips/dts/mt7620a_cameo_810.dtsi

```diff
@@ -163,10 +167,9 @@
 &pcie0 {
 	wifi@0,0 {
 		reg = <0x0000 0 0 0 0>;
-		mediatek,mtd-eeprom = <&factory 0x8000>;
-		ieee80211-freq-limit = <5000000 6000000>;
-		nvmem-cells = <&macaddr_factory_28>;
-		nvmem-cell-names = "mac-address";
+		nvmem-cells = <&eeprom_factory_8000>, <&macaddr_factory_28>;
+		nvmem-cell-names = "eeprom", "mac-address";
 		mac-address-increment = <2>;
+		ieee80211-freq-limit = <5000000 6000000>;
```

Comment:
If it can be accepted, I would like to move all `ieee80211-freq-limit` to the top of the nvmem stuff. It looks a bit ugly.

Reply:
It's ok but please make this change in a different commit to reduce patch delta. (before the conversion)